Automattic/wp-calypso
# Update/remove unused code email providers comparison

This is more or less a follow-up to #64953.

#### Proposed Changes

* Removes unused code for the `<EmailProvidersComparison/>` component.
* Cleans up the paths a bit, and moves the stacked version of the component to a more consistent path to reduce naming confusions.

#### Testing Instructions



* This is mostly just moving files around.
* However we should ensure that we can purchase new Google Workspace and Professional Email subscriptions successfully
* We should also ensure that the domain upsell works i.e. we should be able to purchase an email subscription in addition to a domain during checkout



Related to #64953
